src/routines/write_model.jl

```diff
@@ -0,0 +1,122 @@
+function write_op_model(op_model::OperationModel, path::String)
+    MOF_model = MOPFM
+    MOI.copy_to(MOF_model, JuMP.backend(op_model.canonical.JuMPmodel))
+    MOI.write_to_file(MOF_model, path)
+end
+
+function write_memory_results(results::OperationModelResults, save_path::String)
+
+    new_folder = mkdir("$save_path/$(round(Dates.now(),Dates.Minute))")
+    folder_path = new_folder
+    write_variable_results(results.variables, folder_path) 
+    write_optimizer_results(results.optimizer_log, folder_path)
+    write_time_results(results.times, folder_path)
+    return println("Files written to $folder_path.")
+end
+# taking the outputted files for the variable DataFrame and writing them to a featherfile
+
+function write_variable_results(vars_results::Dict{Symbol, DataFrames.DataFrame}, save_path::AbstractString)
```

Comment:
Why do we have this method if there is a `write_variable_results` already.

Reply:
I'm confused. I only see the one method for write_variable_results? It's called in each of the write results functions. Could you clarify?

Reply:
It looks like we have 2 write results functions. 
1. that takes a canonical. 
2. that takes vars results

Reply:
yes. the one that takes the canonical was already written for the case that nothing is outputted and all results are written straight to feather files (like in a sequential scenario). I wrote the vars_results function so that the variable results can be written to a feather file without the whole simulation running again. Does this make sense? I'm not using the write_results with the canonical_model at all, but I also didn't think I should delete it.